context/app/static/js/components/workspaces/workspaceMessaging.tsx

```diff
@@ -114,51 +140,70 @@ function AccessAlert() {
   );
 }
 
-function TemplateLogInPanel() {
-  if (isAuthenticated) {
-    return null;
-  }
-
-  return (
-    <LogInPanel>
-      Explore workspace templates designed to help you start analyzing HuBMAP data. Use tags to filter templates by your
-      specific interests. Click on any template for detailed information.{' '}
-      <InternalLink href="/login">Log in</InternalLink> to begin working in a workspace.
-    </LogInPanel>
-  );
+interface TemplatePreviewSectionProps {
+  trackingCategory: WorkspacesEventCategories;
 }
 
-function TemplateGridPreview() {
+function TemplatePreviewSection({ trackingCategory }: TemplatePreviewSectionProps) {
   const { selectedItems: selectedRecommendedTags, toggleItem: toggleTag } = useSelectItems([]);
   const [selectedTags, setSelectedTags] = useState<string[]>([]);
   const { templates } = useWorkspaceTemplates([...selectedTags, ...selectedRecommendedTags]);
 
   const recommendedTags = ['visualization', 'api'];
 
-  return (
-    <Stack spacing={2}>
-      <TemplateTagsAutocomplete
-        selectedTags={selectedTags}
-        recommendedTags={recommendedTags}
-        toggleTag={toggleTag}
-        setSelectedTags={setSelectedTags}
-        selectedRecommendedTags={selectedRecommendedTags}
-      />
-      <TemplateGrid templates={templates} />
-    </Stack>
-  );
-}
-
-function TemplatePreviewSection() {
-  if (isAuthenticated) {
-    return null;
-  }
+  const trackLogIn = () => {
+    trackEvent({
+      category: trackingCategory,
+      action: 'Log In / From template section',
+      label: 'template button',
+    });
+  };
 
   return (
-    <Stack pt={2} spacing={2}>
-      <Typography variant="h4">Workspace Templates</Typography>
-      <TemplateLogInPanel />
-      <TemplateGridPreview />
+    <Stack spacing={3}>
+      {!isAuthenticated && (
+        <LogInPanel onClick={trackLogIn}>
+          Explore workspace templates designed to help you start analyzing HuBMAP data. Use tags to filter templates by
+          your specific interests. Click on any template for detailed information.{' '}
+          <InternalLink onClick={trackLogIn} href="/login">
```

Comment:
These both appear to track log-in - i think this will fire two events on click since the link click event will bubble up to the panel container.